ui/packages/catalog-ui-search/src/main/webapp/component/list-interactions/list-interactions.view.js

```diff
@@ -72,9 +74,36 @@ module.exports = Marionette.ItemView.extend({
     const newList = new this.model.constructor(copyAttributes)
     this.model.collection.add(newList)
   },
-  triggerAction(event) {
-    const url = event.currentTarget.getAttribute('data-url')
-    window.open(url)
+  async triggerAction(event) {
+    let url = event.currentTarget.getAttribute('data-url')
+
+    let res = await fetch(url, { Accept: 'application/json' })
```

Comment:
should this await be put into the try catch block?

Reply:
Can fetch calls fail throw an Error?

Reply:
It looks like a reject will throw an error https://javascript.info/async-await#error-handling

Reply:
Ok I will move it in.